.github/workflows/generate_examples_coq.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate Examples - Coq
+
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/cmester0/hacspec_coq:8.16
+      options: --user root
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: |
+        export HOME=/home/coq
+        env
```

Comment:
Is this `env` here for debugging purposes?

Reply:
Yes, to get logging of what is running.